doc/nvim-treesitter.txt

```diff
@@ -415,13 +415,26 @@ to the jump list.
 ==============================================================================
 FUNCTIONS                                          *nvim-treesitter-functions*
 
-                                                *nvim_treesitter#statusline()*
-nvim_treesitter#statusline(size)~
+                                          *nvim_treesitter#statusline()*
+nvim_treesitter#statusline(opts)~
 
-Returns a string describing the current position in the syntax tree. This
+Returns a string describing the current position in the file. This
 could be used as a statusline indicator.
-Note: The `size` argument is optional. When specified, the string will not be
-      longer than `size`.
+Default options (lua syntax):
+>
+  {
+    indicator_size = 100,
+    type_patterns = {'class', 'function', 'method'},
+    transform_fn = function(line) return line:gsub('%s*[%[%(%{]*%s*$', '') end,
```

Comment:
Not sure if we need this now

Reply:
get_node_text will only get the identifier (no () or {})

Reply:
Not for all languages. For example, in typescript class, I get something like this without the transform_fn:
```
class LspClientImpl implements LspClient { -> { -> showMessage(args: lsp.ShowMessageParams): void {
```

And with transform_fn:
```
class LspClientImpl implements LspClient -> showMessage(args: lsp.ShowMessageParams): void
```

Reply:
Oh, I see, this is just navigating the nodes, I thought it was using the query files.